src/audio/AudioWorld.h

```diff
@@ -147,26 +122,16 @@ namespace World
          */
         const VDFS::FileIndex& m_VDFSIndex;
 
-        ALCcontext* m_Context = nullptr;
-
-        Daedalus::DaedalusVM* m_SoundVM = nullptr, *m_MusicVM = nullptr;
-
-        struct Source
-        {
-            unsigned m_Handle = 0;
-            Utils::Ticket<AudioWorld> soundTicket;
-        };
+        Daedalus::DaedalusVM *m_SoundVM = nullptr, *m_MusicVM = nullptr;
 
         struct Sound : public Handle::HandleTypeDescriptor<Handle::SfxHandle>
```

Comment:
I haven't seen virtual dtor of `HandleTypeDescriptor`, isn't there problem with leaking memory? (by vector and string)

Reply:
I don't know the details of the memory management inside the engine, this is something @ataulien or @markusobi know better than me